haystack/nodes/answer_generator/openai.py

```diff
@@ -52,22 +53,24 @@ class OpenAIAnswerGenerator(BaseGenerator):
     def __init__(
         self,
         api_key: str,
-        model: str = "text-curie-001",
+        model: str = "text-davinci-003",
```

Comment:
@sjrl I suggest making this the new default. Or did you find curie to be superior in any experiments?

Reply:
I didn't run any experiments in this case. I agree updating the default here makes sense.